bodhi/server/scripts/manage_releases.py

```diff
@@ -27,17 +27,29 @@
     --pending-testing-tag f23-updates-testing-pending --override-tag f23-override --state pending
 """
 
-import sys
-import munch
-
 import click
 
 from bodhi.client import bindings
+from bodhi.client import (
+    cli,
+    add_options,
+    release_options,
+    create_release,
+    edit_release,
+    info_release
+)
```

Comment:
There may be a simpler way to do this that won't involve importing and wrapping all of these commands. It looks like you can access the ```bodhi releases``` ```Command``` like this:

```
cli.commands['releases']
```

I think something like this might work nicely, and require less code in here:

```
def main():
    """Print a deprecation warning and call the bodhi releases command."""
    click.echo("This utility has been deprecated. Please use 'bodhi releases' instead.")
    cli.commands['releases']()
```